core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -411,14 +411,14 @@ class WorkspaceService(protected val userInfo: UserInfo,
       _ <- requesterPaysSetupService.revokeAllUsersFromWorkspace(workspaceContext) recoverWith {
         case t:Throwable => {
           logger.warn(s"Unexpected failure deleting workspace (while revoking 'requester pays' users) for workspace `${workspaceName}`", t)
-          throw t
+          Future.failed(t)
```

Comment:
good practice to use Future.failed(t) instead of throwing t

Reply:
verified that this still works as before:

```
[WARN] [21:26:36.164] [scala-execution-context-global-48] o.b.d.r.workspace.WorkspaceService - Unexpected failure deleting workspace (while deleting in Workspace Manager) for workspace `gpalloc-qa-master-tpk7jyt/AuthDomainGroupApiSpec_workspace 6168d613-b8e4-454e-9657-b1261b0e0ec4. Received 401: [<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head><script src="https://us.jsagent.tcell.insight.rapid7.com/tcellagent.min.js" tcellappid="FCNonprod-NaVu9" tcellapikey="AQQBBAFLGLOxL7VE9IF9ESlLvCxD5Ykr_7xkQKq_rgn_P58IWjOhOzIh6p3aI4pTWaprlUw" tcellbaseurl="https://us.agent.tcell.insight.rapid7.com/api/v1"></script>
<title>401 Unauthorized</title>
</head><body>
<h1>Unauthorized</h1>
<p>This server could not verify that you
are authorized to access the document
requested.  Either you supplied the wrong
credentials (e.g., bad password), or your
browser doesn't understand how to supply
the credentials required.</p>
<hr>
<address>Apache Server at workspace.dsde-dev.broadinstitute.org Port 443</address>
</body></html>
```

Reply:
This error is from the proxy, not from Sam code

Reply:
sorry, yeah the first line is from sam though, including: `Unexpected failure deleting workspace (while deleting in Workspace Manager) `